src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1454,6 +1454,13 @@ This version used in: {2}</value>
   <data name="Make_member_static" xml:space="preserve">
     <value>Make static</value>
   </data>
+  <data name="ChangeSignature_NewParameterIntroduceTODOVariable" xml:space="preserve">
+    <value>TODO</value>
+    <comment>"TODO" is an indication that there is work still to be done.</comment>
+  </data>
+  <data name="ChangeSignature_NewParameterOmitValue" xml:space="preserve">
+    <value>&lt;omit&gt;</value>
```

Comment:
Is there value in adding `<` `>` around "omit"? 

Also, should it be capitalized? `Omit`

Reply:
I like it as is personally. The `<` `>` are there to indicate that it's _special_, it's not going to put in the word "omit".... you've chosen a special option and I'm trying to show you that. Any better ideas on how to do that would be great too.